src/libraries/Microsoft.Extensions.FileSystemGlobbing/tests/FunctionalTests.cs

```diff
@@ -801,30 +825,38 @@ private void ExecuteAndVerify(Matcher matcher, string directoryPath, params stri
             AssertExtensions.CollectionEqual(expected, actual, StringComparer.OrdinalIgnoreCase);
         }
 
-        // https://github.com/dotnet/runtime/issues/44767
-        [Fact] 
+        [Fact] // https://github.com/dotnet/runtime/issues/44767
         public void VerifyAbsolutePaths_HasMatches()
         {
             var fileMatcher = new Matcher();
             fileMatcher.AddInclude("**/*");
 
-            string fakeWindowsPath = "C:\\This\\is\\a\\nested\\windows-like\\path\\somefile.cs";
-            Assert.True(fileMatcher.Match(Path.GetPathRoot(fakeWindowsPath), fakeWindowsPath).HasMatches);
-
+            if (PlatformDetection.IsWindows)
+            {
+                // Windows-like absolute paths are not supported on Unix.
+                string fakeWindowsPath = "C:\\This\\is\\a\\nested\\windows-like\\path\\somefile.cs";
+                Assert.True(fileMatcher.Match(Path.GetPathRoot(fakeWindowsPath), fakeWindowsPath).HasMatches);
+            }
```

Comment:
Optional: put this in a separate, Windows-specific unit test.

Reply:
I think one test is fine since I am linking the issue reporting the scenario on the top of the method.